zulipterminal/ui_tools/views.py

```diff
@@ -331,18 +331,28 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
 
```

Comment:
You seem to have an extra line, which I'm guessing you don't need, at the bottom of the commit text - was this from a squashed commit?

Reply:
Yup this was from a squashed commit.